src/suit/js/SUIT.js

```diff
@@ -121,6 +126,41 @@ let options = {
         services: tapServices,
         defaultMaxrec: 50000
     },
+    tapObsCore: {
+        [LSST_TAP_LABEL]  : {
+            filterDefinitions: [
+                {
+                    name: 'LSSTCam',
+                    options: [
+                        {label: 'u', value : '367', title: '367nm central value'},
+                        {label: 'g', value : '483', title: '483nm central value'},
+                        {label: 'r', value : '622', title: '622nm central value'},
+                        {label: 'i', value : '755', title: '755nm central value'},
+                        {label: 'z', value : '869', title: '869nm central value'},
+                        {label: 'y', value : '971', title: '971nm central value'},
+                    ],
+                },
+            ],
+            obsCoreCalibrationLevel: {
+                tooltip: 'Calibration ',
+                helptext: '1 is raw data; 2 is PVIs; 3 selects difference images and coadds',
+                level: {
+                    1: {title: 'For Rubin: Raw Data'},
+                    2: {title: 'For Rubin: PVIs'},
+                    3: {title: 'For Rubin: Difference Images and Coadds'},
+                },
+            },
+            obsCoreInstrumentName: {
+                tooltip: 'LSSTCam, LSSTComCam',
+                placeholder: 'e.g. LSSTCam',
```

Comment:
'e.g., LSSTCam-imSim'

Will it be possible to further override just this one thing in deployment configurations?  Or does the whole override from here have to get replaced then?

Reply:
we attempt to substitute the config object like that. I think it would work but we would have to test it.